Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -5402,6 +5513,10 @@ def main():  # pragma: no cover
             results = create_shared_credential_command(client=client, **args)
         elif command == "nexpose-create-site":
             results = create_site_command(client=client, template_id=args.pop("scanTemplateId", None), **args)
+        elif command == "nexpose-get-site-included-targets":
+            results = get_site_included_targets(client=client, **args)
+        elif command == "nexpose-update-site-included-targets":
+            results = update_site_included_targets(client=client, **args)
```

Comment:
Please add these commands to the Integration README. Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/README.md

You can use `demisto-sdk generate-docs` command. https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md